layers/bristol.map

```diff
@@ -0,0 +1,35 @@
+MAP
+  NAME "FMS_WFS_server"
+  STATUS ON
+  SIZE 400 300
+  EXTENT -180 -90 180 90
+  UNITS DD
+  SHAPEPATH "../../layers"
+  IMAGECOLOR 255 255 255
+
+  OUTPUTFORMAT
+    NAME "geojson"
+    DRIVER "OGR/GEOJSON"
+    MIMETYPE "application/json; subtype=geojson"
+    FORMATOPTION "STORAGE=stream"
+    FORMATOPTION "FORM=SIMPLE"
+  END
+
+  LAYER
+    NAME "parks"
+    METADATA
+      "wfs_title"         "Bristol Parks"
+      "wfs_srs"           "EPSG:3857 EPSG:27700"
+      "gml_include_items" "SITE_NAME,SITE_CODE"
+      "gml_featureid"     "SITE_CODE"
+      "wfs_enable_request" "*"
+    END
+    TYPE POLYGON
+    STATUS ON
+    DATA 'bristol/CONTRACT_FEATURES_BySite'
```

Comment:
It doesn't really matter, I guess, but could this have had a better filename? :)

Reply:
The first entry in this file is a multipolygon with a blank site code and name?

Reply:
I'm not really sure...is that something I need to handle? It doesn't sound normal, or is it?

Reply:
I've opened it in QGIS and it looks to be tiny, so probably just one to ignore (but is mistake in their data, we shouldn't have to handle it).